src/ApiService/ApiService/onefuzzlib/VmssOperations.cs

```diff
@@ -159,14 +159,15 @@ public class VmssOperations : IVmssOperations {
             instanceVm.Data.ProtectionPolicy = newProtectionPolicy;
 
             var scaleSet = GetVmssResource(name);
-
-            VirtualMachineScaleSetVmInstanceRequiredIds ids = new VirtualMachineScaleSetVmInstanceRequiredIds(new[] { instanceVm.Data.InstanceId });
-            var updateRes = await scaleSet.UpdateInstancesAsync(WaitUntil.Started, ids);
-
-            //TODO: finish this after UpdateInstance method is fixed
-            //https://github.com/Azure/azure-sdk-for-net/issues/28491
-
-            throw new NotImplementedException("Update instance does not work as expected. See https://github.com/Azure/azure-sdk-for-net/issues/28491");
+            var vmCollection = scaleSet.GetVirtualMachineScaleSetVms();
+            var r = await vmCollection.CreateOrUpdateAsync(WaitUntil.Started, instanceVm.Data.InstanceId, instanceVm.Data);
+            if (r.GetRawResponse().IsError) {
```

Comment:
There's a bit more logic we need to handle here. The `" is not an active Virtual Machine Scale Set VM instanceId."`:

https://github.com/microsoft/onefuzz/blob/3dd27be55b47b1bd9e5a988c217e16f8daf9a22c/src/api-service/__app__/onefuzzlib/azure/vmss.py#L187-L207